src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Unix.targets

```diff
@@ -21,6 +21,8 @@ The .NET Foundation licenses this file to you under the MIT license.
     <CppLinker>$(CppCompilerAndLinker)</CppLinker>
     <CppLibCreator>ar</CppLibCreator>
     <DsymUtilOptions Condition="'$(TargetOS)' == 'OSX'">--flat</DsymUtilOptions>
+    <ObjCopyName Condition="'$(TargetOS)' != 'OSX' and '$(CppCompilerAndLinker)' != 'clang'">objcopy</ObjCopyName>
+    <ObjCopyName Condition="'$(TargetOS)' != 'OSX' and '$(CppCompilerAndLinker)' == 'clang'">llvm-objcopy</ObjCopyName>
```

Comment:
How about the version suffix? On my Linux devbox, I don't have any version of llvm tools (including clang) without the suffix.

Reply:
This is a good point.

Currently, with auto-detection, we only support toolchain selection without the version suffix in NativeAOT. On Ubuntu, for example, when we install clang/llvm from package manager, it installs a package called llvm-defaults, which installs llvm-objcopy (along other tools) in /usr/bin:

```sh
$ command -v llvm-objcopy
/usr/bin/llvm-objcopy
```
However, for versioned `clang` or `gcc`  user can set a property `-p:CppCompilerAndLinker=clang-10`, but for objcopy, we don't have it.

I can add support to set `ObjCopyName` property externally similar to `CppCompilerAndLinker`. But I think it is better if we implement the complete autodetection properly (by packaging `init-compiler.sh`). That is in my TODO list. :)

Reply:
@am11 Thank you for the details, that makes sense. Let's keep this change as is and implement the version detection separately.

Reply:
Modulo @jkotas's comment, so let's add the ObjCopyName

Reply:
I have changed it so it is set conditionally. `-p:ObjCopyName=llvm-objcopy-10` or `-p:ObjCopyName=/path/to/llvm-objcopy`.